onnx_chainer/replace_func.py

```diff
@@ -0,0 +1,129 @@
+import inspect
+from inspect import Parameter
+
+import chainer
+
+
+class WrappedFunctionNode(chainer.FunctionNode):
+
```

Comment:
Could you add a brief docstring for this? A very short one should be fine since users won't use this directly.

Reply:
fixed